<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWindow</class>
 <widget class="QMainWindow" name="TestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 16pt &quot;Cantarell&quot;;</string>
    </property>
    <property name="text">
     <string>Test for Date   : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="testDate">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Cantarell&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>Question 1: Whom have you met on on the mentioned date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Q1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>Question 2: What have you eaten on the mentioned date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Q2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>Question 3: Where did you go on the mentioned date </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Q3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Cantarell&quot;;
background-color: rgb(53, 53, 53);
</string>
    </property>
    <property name="text">
     <string>Submit Test</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>Question 4: Birthdays (if any)  on on the mentioned date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Q4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>Question 5: Aniversaries (if any)  on on the mentioned date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Q5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
